### DISCUSSION TITLE
How to preview the searching post (Plural post)

### DISCUSSION THREAD

**Discussion by stephenwongcandr** [wp-graphql/wp-graphql#2470] [Q&A] [answered]
### What problem does this address?

I have a ACF field with CPT UI name shop (example)
GraphQL Single Name shop
GraphQL Plural Name shops
and the front-end is using the query shops to get all the shop posts, is that possible to get the listing post even the post is draft?
currently, shops queries only return all published post

### What is your proposed solution?

I want the shops query(Plural) can add asPreview function, when asPreview is true, it will return all the posts including drafts or pending review posts. just like the single post query.

**Comment by justlevine** [comment, marked as answer]
Hey @stephenwongcandr .

You can use the `status` or `statii` where arg to filter connections by the post status.

E.g.
```graphql
shops( where: { stati: [ PENDING, DRAFT, PUBLISH ] } ) {
    nodes {
      databaseId
      status
    }
  }
```

**Reply by stephenwongcandr** [reply]
thanks, it's helpful
